trunner/target/armv7a9.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, dut: Dut, script: str, cwd: Optional[str] = None):
         PloInterface.__init__(self, dut)
         self.script = script
         self.cwd = cwd
-        self.gdbserver = JLinkGdbServer("Zynq 7020")
+        self.gdbserver = OpenocdGdbServer(board="digilent_zedboard")
```

Comment:
This change further limited targets supported by this class to zedboard only thus I feel it should be renamed to `ZynqZedboardGdbPloLoader`

Reply:
I think we could rename it to avoid misunderstandings.